integ-test/src/test/java/org/opensearch/sql/jdbc/CursorIT.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.jdbc;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_ACCOUNT;
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_BANK;
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_CALCS;
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_ONLINE;
+import static org.opensearch.sql.legacy.plugin.RestSqlAction.QUERY_API_ENDPOINT;
+import static org.opensearch.sql.util.TestUtils.getResponseBody;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.List;
+import javax.annotation.Nullable;
+import com.carrotsearch.randomizedtesting.annotations.ThreadLeakScope;
+import lombok.SneakyThrows;
+import org.json.JSONObject;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.opensearch.client.Request;
+import org.opensearch.client.RequestOptions;
+import org.opensearch.client.Response;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
+public class CursorIT extends SQLIntegTestCase {
+
+  private static Connection connection;
+  private boolean initialized = false;
+
+  @BeforeEach
+  @SneakyThrows
+  public void init() {
+    if (!initialized) {
+      initClient();
+      resetQuerySizeLimit();
+      loadIndex(Index.BANK);
+      loadIndex(Index.CALCS);
+      loadIndex(Index.ONLINE);
+      loadIndex(Index.ACCOUNT);
+      initialized = true;
+    }
+  }
+
+  @BeforeAll
+  @BeforeClass
+  @SneakyThrows
+  public static void initConnection() {
+    connection = DriverManager.getConnection(getConnectionString());
+  }
+
+  @AfterAll
+  @AfterClass
+  @SneakyThrows
+  public static void closeConnection() {
+    // TODO should we close Statement and ResultSet?
```

Comment:
Was this answered or was this left on purpose for us to see?

Reply:
It is not answered
I suppose we can leave things unclosed **in tests**.